lib/helper/Appium.js

```diff
@@ -0,0 +1,839 @@
+          or
+          {
+            "helpers": {
+              "WebDriverIO": {
+                desiredCapabilities: {
+                    platformName: "Android",
+                    browserName: "Chrome",
+                    platformVersion: "6.0.1"
+                }
+              }
+            }
+          }
+        `);
+      }
+    }
+
+  }
+
+
+  _startBrowser() {
+    if (this.options.multiremote) {
+      this.browser = webdriverio.multiremote(this.options.multiremote).init();
+    } else {
+      this.browser = webdriverio.remote(this.options).init();
+    }
+
+    if (this.options.timeouts && browserCap) {
+      this.defineTimeout(this.options.timeouts);
+    }
+
+    if (this.options.windowSize === 'maximize') {
+      this.browser.windowHandleMaximize(false);
+    }
+    if (this.options.windowSize && this.options.windowSize.indexOf('x') > 0) {
+      let dimensions = this.options.windowSize.split('x');
+      this.browser.windowHandleSize({
+        width: dimensions[0],
+        height: dimensions[1]
+      });
+    }
+    return this.browser;
+  }
+
+  _after() {
+    if (this.options.restart) return this.browser.end();
+    if (this.options.desiredCapabilities.browserName) {
+      this.debugSection('Session', 'cleaning cookies and localStorage');
+      return this.browser.execute('localStorage.clear();').then(() => {
+        return this.browser.deleteCookie();
+      });
+    }
+  }
+
+  /**
+   * Check if an app is installed.
+   * Appium: support only Android
+   */
+  checkAppIsInstalled(bundleId) {
+    if (this.options.desiredCapabilities.platformName == "Android") {
+      return this.browser.isAppInstalled(bundleId).then(function(res) {
+        return truth(`app ${bundleId}`, 'to be installed').assert(res.value);
+      });
+    } else
+      throw new Error(`checkAppInstallation method can be used only with Android apps`);
+  }
+
+  /**
+   * Check if an app is not installed.
+   * Appium: support only Android
+   */
+  checkAppIsNotInstalled(bundleId) {
+    if (this.options.desiredCapabilities.platformName == "Android") {
+      return this.browser.isAppInstalled(bundleId).then(function(res) {
+        return truth(`app ${bundleId}`, 'to be installed').negate(res.value);
+      });
+    } else
+      throw new Error(`checkAppInstallation method can be used only with Android apps`);
+  }
+
+  /**
+   * check current activity on an Android device.
+   * Appium: support only Android
+   */
+  checkCurrentActivityIs(currentActivity) {
+    if (this.options.desiredCapabilities.platformName == "Android") {
+      return this.browser.currentActivity().then(function(res) {
+        return truth('current activity', `to be ${currentActivity}`).assert(res.value == currentActivity);
+      });
+    } else
+      throw new Error(`getCurrentActivity method can be used only with Android apps`);
+  }
+
+  /**
+   * Check whether the device is locked.
+   * Appium: support only Android
+   */
+  checkDeviceIsLocked() {
```

Comment:
if it is an assertions (and it is) the method should start with `see`: `seeDeviceIsLocked`